omnibus/files/private-chef-cookbooks/private-chef/recipes/redis_lb.rb

```diff
@@ -111,14 +111,18 @@
                       port: redis_data['port'],
                       password: PrivateChef.credentials.get('redis_lb', 'password'))
     xdl = node['private_chef']['lb']['xdl_defaults']
+    xmaint_whitelisted_ip = node['private_chef']['lb']['xmaint_whitelisted_ip']
```

Comment:
Question: Does this represent a single IP address, or multiple IPs?  If multiple, I'd consider changing the name to xmaint_whitelisted_ips (add an 's' to the end).  I noticed you used an 'each' iterator on this later, so maybe it's multiple?

Reply:
It represents list of ips. I have changed the name to 'xmaint_allowed_ips_list'